### ISSUE TITLE
error: Not found: No attr named 'T' in NodeDef when running lammps

### ISSUE THREAD

**Issue by minghuisvn** [deepmodeling/deepmd-kit#416]
I tried the water data in the deepmd-kit' repository example folder, and obtained the frozened model, graph.pb. But when running lammps by "lmp -i in.lammps", the errors occured. The error message was:
```
lmp: Relink `/home/linazhao/anaconda3/envs/dpmd/bin/../lib/./libgfortran.so.4' with `/lib/x86_64-linux-gnu/librt.so.1' for IFUNC symbol `clock_gettime'
2021-03-15 13:27:39.025473: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
LAMMPS (29 Oct 2020)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:94)
Warning: please export TSAN_OPTIONS='ignore_noninstrumented_modules=1' to avoid false positive reports from the OpenMP runtime!
  using 1 OpenMP thread(s) per MPI task
Reading data file ...
  triclinic box = (0.0000000 0.0000000 0.0000000) to (12.444700 12.444700 12.444700) with tilt (0.0000000 0.0000000 0.0000000)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  192 atoms
  read_data CPU = 0.001 seconds
Summary of lammps deepmd module ...
  >>> Info of deepmd-kit:
  installed to:       /home/linazhao/anaconda3/envs/dpmd
  source:             v1.3.2
  source brach:       HEAD
  source commit:      2644cca
  source commit at:   2021-03-02 06:30:13 +0800
  build float prec:   double
  build with tf inc:  /home/linazhao/anaconda3/envs/dpmd/include;/home/linazhao/anaconda3/envs/dpmd/include
  build with tf lib:  /home/linazhao/anaconda3/envs/dpmd/lib/libtensorflow_cc.so;/home/linazhao/anaconda3/envs/dpmd/lib/libtensorflow_framework.so
  set tf intra_op_parallelism_threads: 0
  set tf inter_op_parallelism_threads: 0
  >>> Info of lammps module:
  use deepmd-kit at:  /home/linazhao/anaconda3/envs/dpmd
  source:             v1.3.2
  source branch:      HEAD
  source commit:      2644cca
  source commit at:   2021-03-02 06:30:13 +0800
  build float prec:   double
  build with tf inc:  /home/linazhao/anaconda3/envs/dpmd/include;/home/linazhao/anaconda3/envs/dpmd/include
  build with tf lib:  /home/linazhao/anaconda3/envs/dpmd/lib/libtensorflow_cc.so;/home/linazhao/anaconda3/envs/dpmd/lib/libtensorflow_framework.so
2021-03-15 13:27:39.160914: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 AVX512F FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2021-03-15 13:27:39.161764: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2021-03-15 13:27:39.476767: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties:
pciBusID: 0000:17:00.0 name: GeForce RTX 2070 SUPER computeCapability: 7.5
coreClock: 1.8GHz coreCount: 40 deviceMemorySize: 7.79GiB deviceMemoryBandwidth: 417.29GiB/s
2021-03-15 13:27:39.477166: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 1 with properties:
pciBusID: 0000:65:00.0 name: GeForce RTX 2070 SUPER computeCapability: 7.5
coreClock: 1.8GHz coreCount: 40 deviceMemorySize: 7.79GiB deviceMemoryBandwidth: 417.29GiB/s
2021-03-15 13:27:39.477186: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-03-15 13:27:39.478532: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-03-15 13:27:39.479767: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-03-15 13:27:39.480076: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-03-15 13:27:39.481273: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-03-15 13:27:39.481847: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-03-15 13:27:39.484181: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-03-15 13:27:39.485599: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0, 1
2021-03-15 13:27:39.485626: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-03-15 13:27:40.234693: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-03-15 13:27:40.234741: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 1
2021-03-15 13:27:40.234747: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N N
2021-03-15 13:27:40.234751: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 1:   N N
2021-03-15 13:27:40.236264: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 7184 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2070 SUPER, pci bus id: 0000:17:00.0, compute capability: 7.5)
2021-03-15 13:27:40.237286: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 7181 MB memory) -> physical GPU (device: 1, name: GeForce RTX 2070 SUPER, pci bus id: 0000:65:00.0, compute capability: 7.5)
Not found: No attr named 'T' in NodeDef:
         [[{{node DescrptSeA}}]]
         [[DescrptSeA]]
```
the deepmd version is v.1.3.2.

**Comment by amcadmus** [comment]
The model was trained with a DeePMD-kit version < 1.3.
